contents/blog/how-to-find-and-fix-onboarding-drop-off.mdx

```diff
@@ -0,0 +1,445 @@
+
+If everyone drops off at the same step, it's probably a UX problem. Something about that step is broken or confusing for all users.
+
+If only *some* users drop off, it's a context problem. Something about who they are or how they got there is causing friction.
+
+If it's a context problem, try segmenting your funnel to find a clearer diagnosis:
+- New users vs. returning users
+- Invited users vs. self-serve signups
+- Browser, device, or OS
+- Plan type or pricing tier
+
+<details>
+<summary><strong>If using PostHog</strong></summary>
+
+Click on the drop-off number in your funnel to see the actual users who didn't make it. You can save these as a [cohort](https://posthog.com/docs/data/cohorts) for further analysis – useful for targeting with surveys later, or watching their replays in Step 5.
+
+Use [breakdowns](https://posthog.com/docs/product-analytics/funnels#breakdowns) to slice your funnel by user properties, device, or any event property.
+
+</details>
+
+By the end of this step, you should be able to say: "Users drop off most at *[this step]*" and "It affects *[these users]* more than others." If you can't say both, don't move on yet.
+
+If you do, you're ready for the fun part: watching what actually happens.
+
+---
+
+## Step 5: Watch the replays
+
+You know where users drop off, [now you need to find out *why*](https://posthog.com/tutorials/explore-insights-session-recordings).
+
+Watch 10–15 recordings. You're looking for patterns:
+- Are users getting confused at a specific UI element?
+- Are they rage-clicking something that doesn't work?
+- Are they abandoning after seeing a specific screen (pricing, permissions request, etc.)?
+- Are they hitting errors? (Check the console logs in the replay; PostHog [captures these](https://posthog.com/docs/session-replay/console-log-recording) too.)
+- Is something failing quietly? Look at network requests if you have [network recording](https://posthog.com/docs/session-replay/network-recording) enabled.
+
+If recordings are looking wildly different from one user to the next, go back to Step 4 and segment further. You're probably mixing multiple problems together. Cleaner inputs make cleaner diagnoses.
+
+<details>
+<summary><strong>If using PostHog</strong></summary>
+
+**Fastest way:** Click directly on the drop-off in your funnel – PostHog will pull up recordings for those users automatically. (This is one of the nice things about having replay and analytics in one tool!)
+
+**Manual way:** In [Session Replay](https://app.posthog.com/replay), click **Show filters** → **Filter for events or actions** → select the last event users completed before dropping off. ([Session replay filtering guide](https://posthog.com/tutorials/filter-session-recordings))
+
+If you saved a cohort in Step 4, you can filter replays by that cohort directly.
+
+</details>
+
+By the end of Step 5, you should be able to say "Users drop off here because *[this thing]* keeps happening."
```

Comment:
Again I think consistent format for checking how you know you've d1one this step successfully (maybe a quote block box or callout box with consistent format) will be super useful